app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, html: {'data-turbo' => "false"}, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ ] Same here. I would suggest following a design. 

### Your design 

![Screenshot from 2022-03-25 03-41-41](https://user-images.githubusercontent.com/22434660/160019526-a616d80b-2480-4691-aad7-853cf9534012.png)

### Given design

![Screenshot from 2022-03-25 03-49-23](https://user-images.githubusercontent.com/22434660/160019570-9459e36f-9840-4450-b8ce-3d8a74cbed73.png)

Reply:
This issue is well resolved! ✔️